src/accessibility/KeyboardShortcuts.ts

```diff
@@ -711,4 +715,14 @@ export const KEYBOARD_SHORTCUTS: IKeyboardShortcuts = {
             key: Key.COMMA,
         },
     },
+    [KeyBindingAction.PageUp]: {
+        default: {
+            key: Key.PAGE_UP,
+        },
+    },
+    [KeyBindingAction.PageDown]: {
+        default: {
+            key: Key.PAGE_DOWN,
+        },
+    },
```

Comment:
Why aren't we reusing `KeyBindingAction.ScrollUp` and `KeyBindingAction.ScrollDown`? More of a question than a comment, it may make sense to split them, just wondering.

Reply:
KeyBindingAction.ScrollUp and KeyBindingAction.ScrollDown are specifically bind to timeline scroll If we use that then we will have those display names also that are bind with them [here](https://github.com/matrix-org/matrix-react-sdk/blob/935889e64b1f73f99e32f85693376ce01a0ced9d/src/accessibility/KeyboardShortcuts.ts#L454).